### DISCUSSION TITLE
How often do you use `t<char>` & `f<char>` selections?

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#4025] [Polls]
`t<char>`, `f<char>`, `T<char>`, & `F<char>` are frequently used for movement. They select the text on movement. How often do you see yourself cancelling those selections?

So basically, are those selections mostly useful to you or do you cancel them more often.

**Comment by CptPotato** [comment]
Interesting point. Most of the time I use `f` when I want to insert something before or after a certain character like adding an additional parameter to a function: `f);i`.

In this case the selection is actually counter productive. Not sure if it's currently possible remap the command to collapse the selection afterwards because it requires an additional input.
